soccer/gameplay/tactics/stopped/circle_near_ball.py

```diff
@@ -155,11 +139,27 @@ def execute_running(self):
         for b in self.all_subbehaviors():
             if b.robot is not None:
                 num_robots += 1
+        #if the number of robots has changed, recreate move behaviors to match new number of robots
+        if (self.num_robots != num_robots):
+            self.num_robots = num_robots
+            self.remove_all_subbehaviors()
+            i = 0
+            for pt in range(6):
+                self.add_subbehavior(skills.move.Move(),
+                                     name="robot" + str(i),
+                                     required=False,
+                                     priority=6 - i)
+                i = i + 1
 
         i = 0
+        #assign destinations for the number of robots we have
         for pt in self.get_circle_points(num_robots):
             self.subbehavior_with_name("robot" + str(i)).pos = pt
             i = i + 1
+        #unassign destinations from behaviors without robots
+        for i in range(num_robots, 6):
+            self.subbehavior_with_name("robot" + str(i)).pos = None
```

Comment:
Oh probably because the play starts with no robots, so if it never adds any behaviors (because it keeps remove them) it will never get any robots and nothing will happen. It needs behaviors with no position to get robots